src/js/fluxcontroller.js

```diff
@@ -105,6 +105,8 @@ define(function (require, exports, module) {
         this._actionQueue = new AsyncDependencyQueue(CORES);
         this._initActionNames();
         this._initActionLocks();
+        this._sychronizedActions = new Map();
```

Comment:
This is not declared in `FluxController.prototype` or anywhere, and is lacking JSDoc.